.circleci/config.yml

```diff
@@ -8,7 +8,9 @@ jobs:
 
       # Install development dependencies
       - run: sudo apt-get install python3 python3-pip
-      - run: sudo npm install -g jsonlint eslint flow flow-bin flow-coverage-report htmlhint sass-lint
+      - run: sudo npm install -g jsonlint eslint flow flow-bin flow-coverage-report htmlhint sass-lint grunt-cli
```

Comment:
We should just have a `package.json` at this point.

Reply:
@sushain97 I tried for several hours to try to get this to work, but I was unable to do this. I don't exactly know, but it might be because of the `-g`, and apparently installing “global” npm dependencies via package.json isn't supported? I created the `package.json` and was running `npm install`, but I kept on getting `No such file or directory`.

Reply:
Looks like this has since been done in latest master!